os/include/tinyara/mm/mm.h

```diff
@@ -249,6 +249,15 @@ typedef size_t mmsize_t;
 #error Unknown CONFIG_ARCH option, malloc debug feature wont work.
 #endif
 
+#ifdef CONFIG_DEBUG_MM_HEAPINFO
+/* Macro sets caller address of malloc API to mem chunk.
+ * If rapping malloc APT, have to called after calling the malloc API.
```

Comment:
How about below?

This macro updates a caller return address in the 'mem' node.
This will show the real owner of the `mem` even it's allocated through wrapping APIs of malloc.

Reply:
Thank you. I modified comment.